src/Concerns/Internal/FileHelpers.php

```diff
@@ -92,7 +105,7 @@ public static function relativeLink(string $destination, string $current = ''):
         if ($nestCount > 0) {
             $route .= str_repeat('../', $nestCount);
         }
-        $route .= $destination;
+        $route .= static::pageLink($destination);
 
         return $route;
```

Comment:
Handles edge case encountered in footer of sidebar documentation linking to index page using pretty URLs where the result became `..//` instead of `../`